README.md

```diff
@@ -120,11 +120,51 @@ luatest.helpers.retrying({timeout = 1, delay = 0.1}, failing_function, arg1, arg
 luatest.helpers.retrying({}, function() server:http_request('get', '/status') end)
 ```
 
+## List of luatest functions
```

Comment:
Please, move that section in the **Usage** section

Reply:
This will move section with server helpers much further. Don't you think server-helper usage should go right after usage and then the list of available functions?

Reply:
Yes, your point is curious. Maybe we should create the table of contents at the footer of README. It will briefly notify the user, which will observe the repo.

Reply:
Well, I moved this section to the top for now.